components/HomePage/index.tsx

```diff
@@ -1,10 +1,153 @@
-import React from "react";
+import React, { useMemo } from "react";
 import { DefaultLayout } from "../../layouts/Default";
+import { PrimaryLink, SecondaryLink } from "../Link";
+import SeoHead from "../SeoHead";
+import * as UI from "./components";
+import { constants } from "../../utils";
+import { useHomePage } from "./useHomePage";
+import { UMA_FEATURES } from "./features";
+import { PRODUCTS } from "./products";
+import { OOBoxesData } from "./oo";
+
+const { LINKS, PROJECTS } = constants;
 
 export const HomePage: React.FunctionComponent = () => {
   return (
     <DefaultLayout>
-
+      <SeoHead />
+      <UI.Container>
+        <UI.Content>
+          <UI.Title>UMA is an <span>Optimistic Oracle</span> built for Web3</UI.Title>
+          <PrimaryLink href={LINKS.docs}>Get started</PrimaryLink>
+        </UI.Content>
+      </UI.Container>
+      <WhatIsOO />
+      <Projects />
+      <WhyUseUMA />
+      <Products />
+      <GetStarted />
     </DefaultLayout>
+  )
+}
+
+const WhatIsOO: React.FunctionComponent = () => {
+  return (
+    <UI.HomeSectionContainer grayBackground>
+      <UI.HomeSectionContent>
+        <UI.HomeSectionHeadline>What is the optimistic oracle?</UI.HomeSectionHeadline>
+        <UI.OODescription>
+          An optimistic oracle services data to smart contracts by assuming it is true unless it is disputed.
+        </UI.OODescription>
+        <UI.OOBoxes>
+          {OOBoxesData.map((data, idx) => (
+            <UI.OOBoxContainer key={idx}>
+              <UI.OOBox>
+                <UI.BoxImgContainer>
+                  <UI.BoxImg src={data.icon} alt="Optimisc Oracle service" />
+                </UI.BoxImgContainer>
+                <UI.OOBoxTextContainer>
+                  <UI.BoxTitle>{data.title}</UI.BoxTitle>
+                  <UI.BoxDescription>{data.description}</UI.BoxDescription>
+                </UI.OOBoxTextContainer>
+              </UI.OOBox>
+            </UI.OOBoxContainer>
+          ))}
+        </UI.OOBoxes>
+        <SecondaryLink href={LINKS.oo}>Learn more</SecondaryLink>
+      </UI.HomeSectionContent>
+    </UI.HomeSectionContainer>
+  );
+}
+
+const Projects: React.FunctionComponent = () => {
+  const { isTablet } = useHomePage();
+  const projects = useMemo(() => isTablet ? PROJECTS : PROJECTS.slice(0, 6), [isTablet]);
```

Comment:
Nit: If we want to follow the new dapp design , we should move this inside useHomePage

Reply:
Moved it 👍